### DISCUSSION TITLE
New version of SC does not keep user data / settings from old one

### DISCUSSION THREAD

**Discussion by archwaycommons** [AAndyProgram/SCrawler#140] [Q&A] [answered]
Hello,

I have SCcralwer version 2022.11.16.0 and when trying to update to 2023..9.0 none of my user data / settings are there. Does the app automatically update itself or do I have to drag & drop the new version into the old versions folder and overwrite it?

**Comment by AAndyProgram** [comment, marked as answer]
https://github.com/AAndyProgram/SCrawler#updating

Unpack SCrawler to the folder where it is placed. This is a portable program.
